src/com/gh4a/loader/IssueCommentListLoader.java

```diff
@@ -51,13 +51,14 @@ public IssueCommentListLoader(Context context, String repoOwner, String repoName
                 new RepositoryId(mRepoOwner, mRepoName), mIssueNumber);
         List<IssueEvent> events = issueService.getIssueEvents(mRepoOwner, mRepoName, mIssueNumber);
         List<IssueEventHolder> result = new ArrayList<>();
+        boolean isPullRequest = this instanceof PullRequestCommentListLoader;
```

Comment:
I don't like base classes having to know about descendant classes. I think I'd prefer a constructor parameter.